index.js

```diff
@@ -447,6 +447,24 @@ Peer.prototype.getStats = function (cb) {
       })
       cb(items)
     }, function (err) { self._onError(err) })
+}
+
+// Detect the WebRTC implementation. We only need this in a few places where
+// feature-detection does not work correctly.
+Peer.prototype._browserDetect = function () {
```

Comment:
Using a `switch (true)` statement for the detection might be better here in case there are intentional fall-through cases where more than 1 condition might result in the same case. Just a suggestion.